### ISSUE TITLE
Create the layout for the main page

### ISSUE THREAD

**Issue by parzhitsky** [public-chat-team/public-chat#14]
Create the layout for the main page.

> See [**[Sketch] Main page**](https://github.com/public-chat-team/public-chat/wiki/%5BSketch%5D-Main-page).

- [ ] the layout is flexible; it looks basically the same on all screens of all sizes;
- [ ] the layout always takes 100% of both viewport width and height; there are no scrollbars around the page;
- the layout may impose reasonable lower limits on viewport size (e.g., `min-width: 400px`); in this case, scrollbars are allowed on smaller screens;
- [ ] the layout must be implemented as a React component, to support other layouts in the future, if needed;

**Comment by parzhitsky** [comment]
Blocked by #15

**Comment by Fidelya1988** [comment]
**the layout always takes 100% of both viewport width and height; there are no scrollbars around the page;**
Should there be no scrollbars in height either?
 **the layout must be implemented as a React component, to support other layouts in the future, if needed;**
You mean that this layout can be used for other components?

**Comment by parzhitsky** [comment]
> Should there be no scrollbars in height either?

Yes, the component must represent the layout of **the whole page** at once. If the page dimensions change for some reason (e.g., the browser window is resized), the size of the component must change too.

>  You mean that this layout can be used for other components?

No, I mean that we might have other pages in the future (for example, user page), and they will definitely use different layouts. Therefore, this layout is unique to this page, and it should be encapsulated inside of a component. This will allow encapsulate other pages in other components, and connect all the application pages in a single router more easily.

**Comment by Fidelya1988** [comment]
and messages will scroll?

**Comment by parzhitsky** [comment]
Didn’t get the question, sorry; please, see the [sketch](https://github.com/public-chat-team/public-chat/wiki/%5BSketch%5D-Main-page)
